x/burn/internal/keeper/burn.go

```diff
@@ -0,0 +1,57 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	accountExported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+)
+
+func (k Keeper) BurnCoins(ctx sdk.Context, acc string) sdk.Error {
+	bunAccount, err := getAccount(ctx, k, acc)
+	if err != nil {
+		return sdk.ErrInvalidAddress(err.Error())
+	}
+
+	if bunAccount == nil {
+		return nil
+	}
+
+	amt := bunAccount.GetCoins()
+
+	if amt.Empty() {
```

Comment:
In Go, we didn't used to put any empty line before the subsequent `if` statement.
```suggestion
	amt := bunAccount.GetCoins()
	if amt.Empty() {
```